### ISSUE TITLE
restructure the Google Analytics events we create for our "doodle" experiment

### ISSUE THREAD

**Issue by escattone** [mozilla/sumo#1436]
I discovered that our Google Analytics (GA) doodle events aren't useful as they're currently structured, because we're embedding the fact of shown vs. not-shown in the GA event's value. GA expects that to be an integer and so only provides cumulative, numerical statistics for it.

**Comment by emilghittasv** [comment]
Hey @escattone !

Unfortunately I don't have access to SUMO analytics. Is this QA testable?

**Comment by escattone** [comment]
@emilghittasv No, but could you help me with one thing? Would you please load the https://support.allizom.org/en-US/kb/switching-devices page a bunch of times, making sure to get both cases covered, the doodle shown and the doodle not shown? For some reason, my visits to that page aren't showing up in GA, so it would be nice to have someone else hit that page as well. I just want to confirm that the new `doodle-not-shown` event is showing-up in GA.

**Comment by emilghittasv** [comment]
@escattone 

I've loaded the https://support.allizom.org/en-US/kb/switching-devices a couple of times. Each time before loading the website I've made sure to clear my cookies and site data so the `dwf_show_device_migration_doodle` cookie gets freshly generated each time (I've covered both `False` and `True` states).

I'm wondering, when you've loaded the https://support.allizom.org/en-US/kb/switching-devices page was the wizard disabled or enabled? I'm just wondering if that has something to do with GA data not being generated for your vists. I've tested this after adding `https://support.allizom.org` as a string value to the  `browser.uitour.testingOrigins` browser pref.
